docs/admin/release_notes/version_2.2.md

```diff
@@ -0,0 +1,6 @@
+
+## [v2.2.0 (2024-06-28)](https://github.com/nautobot/nautobot-app-floor-plan/releases/tag/v2.2.0)
+
+### Added
+
+- [#10](https://github.com/nautobot/nautobot-app-floor-plan/issues/10) - Added the ability to allow a rackgroup or status tile to span multiple floortiles. Additionally rack tiles can be place on top of a group of status or rackgroup tiles.
```

Comment:
I think this file gets generated automatically during the release process from town crier with the proper date and such. Not sure what would happen if it is already created, so I would probably just err on the side of caution and remove it for now.